src/App.js

```diff
@@ -23,7 +23,7 @@ class App extends React.Component {
             <h1>Bookstore CMS</h1>
             <ul className="navigation-list">
               <li>
-                <Link to="/books">BOOKS</Link>
```

Comment:
The user is unable to see all the books added to the list when clicking on `BOOKS`.

Reply:
Hello Sir, I checked on your recommendation, which is a good one. But I have some doubts about it. First, in my project, I can not find some of your screenshots actions(because I changed some codes you're talking about)  Second, I tried to implement some changes you advise me! For now, once you add a book, that book will be automatically added to the list of books and you can see it by clicking on the `BOOKS`, you don't need to refresh the page to see the update, just by clicking on `BOOKS`
![image](https://user-images.githubusercontent.com/71874425/147386293-e91abaf2-cd5e-4052-88b6-3a2c6713062d.png)

As you can see in the pic above, I used the API provided for this project! Check the screenshot of the redux Devtools above, you can see all keys that are mandatory and the unique key for id(which is corresponding to item_id related to my code).
Thanks, Merry Christmas!